<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clock Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	color: rgb(255, 255, 255);
	background-color: rgba(10, 10, 10, 100);
}

QRadioButton::indicator {
    width: 13px;
    height: 13px;
}

QLabel {
	background-color: rgb(0, 0, 0);
	border-color: rgb(10, 10, 10);
	color: rgb(255, 255, 255);
}

QPushButton {
	color: rgb(10, 10, 10);
	background-color: rgb(214, 214, 27);
	border-style: solid;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: rgb(148, 139, 19);
}

QPushButton:hover {
	border: 1px solid rgb(160, 201, 255);
 	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="analog_clock">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Analog clock</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hour24_clock">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>24 hour clock</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="hour12_clock">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>12 hour clock</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="select_clock_but">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>94</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select clock, please</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
